### PR TITLE
Wrote script to perform Sentiment analysis on a column

### PR DESCRIPTION
**Pull request by ayahusseini**
- Performs sentiment analysis on a text column in a dataframe
- assumes cleaned data
- also includes a method to aggregate by source,topic grouping 
- unit tests included

### CODE REVIEW

File: news_sentiment_analyser/test_sentiment_analysis.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+    assert pytest.approx(sentiments['pos'][0], rel=1e-4) == 0.6369
+    assert pytest.approx(sentiments['neg'][1], rel=1e-4) == 0.4939
+
+
+def test_get_sentiments_type_error():
+    """Tests that a TypeError is raised if the 'text' column
+    contains non-string objects"""
+
+    df_non_string = pd.DataFrame({
```
Comment (YakMan101) [inline news_sentiment_analyser/test_sentiment_analysis.py:61]: Might be worth adding this as a fixture or global variable if it is not going to be changed

File: news_sentiment_analyser/test_sentiment_analysis.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+import pytest
+import pandas as pd
+from unittest.mock import patch, MagicMock
+
+from sentiment_analysis import get_sentiments, get_avg_sentiment
+
+
+@pytest.fixture
+def mock_sia():
```
Comment (YakMan101) [inline news_sentiment_analyser/test_sentiment_analysis.py:11]: Nice use of fixture

File: news_sentiment_analyser/test_sentiment_analysis.py (hunk 1 above)
Comment (ayahusseini) [reply news_sentiment_analyser/test_sentiment_analysis.py:61]: Good point - will refactor it into a fixture.

File: news_sentiment_analyser/test_sentiment_analysis.py [hunk 3]
```diff
@@ -7,6 +7,16 @@
 from sentiment_analysis import get_sentiments, get_avg_sentiment
 
 
+@pytest.fixture
```
Comment (indiah444) [inline news_sentiment_analyser/test_sentiment_analysis.py:10]: Good thinking in making this a fixture, reduces amount of repeated code :)

**Review by indiah444 — APPROVED** [review]

**Pull request merged**